docs/dev_guide/documentation.rst

```diff
@@ -114,10 +113,16 @@ This will generate HTML documentation for SunPy in the "docs/_build/html" direct
 You can open the "index.html" file to browse the final product.
 The gallery examples are located under "docs/_build/html/generated/gallery".
 Sphinx builds documentation iteratively, only adding things that have changed.
-If you'd like to start from scratch then just run::
+
+If you'd like to start from scratch (i.e., remove the tox cache) then just run::
```

Comment:
```suggestion
If you'd like to start from scratch (i.e., remove the tox cache) then run::
```
"just" doesn't add much to it and makes it look as it's easy to understand what's going on.